doc/user_guide/user_guide.md

```diff
@@ -116,6 +116,16 @@ We support the Parquet, Avro and Orc formats when importing data from cloud
 storages into an Exasol table. However, we export Exasol tables only as Parquet
 data to storage systems.
 
+### CSV Format Not Supported
+
+In this integration project, we do not support `CSV` format. However, Exasol
+provides optimized `CSV` loader. Please check the [Exasol CSV loader official
+documentation](https://docs.exasol.com/db/latest/sql/import.htm).
+
+We recommend to use the `CSV` loader if your data in the `CSV` format, since
+importing it is optimized and it is substantially fast compared to binary
+formats such as `Parquet`, `Avro` or `ORC`.
```

Comment:
```suggestion
We recommend to use the `CSV` loader if your data is in the `CSV` format, since
importing it is optimized and it is substantially faster compared to binary
formats such as `Parquet`, `Avro` or `ORC`.
```